src/adminsystem.cpp

```diff
@@ -138,6 +138,12 @@ CON_COMMAND_CHAT_FLAGS(ban, "ban a player", ADMFLAG_BAN)
 	if (pTargetPlayer->IsFakeClient())
 		return;
 
+	if (!pTargetPlayer->IsAuthenticated())
+	{
+		ClientPrint(player, HUD_PRINTTALK, CHAT_PREFIX "%s is not yet authenticated, please wait a moment and try again.", pTarget->GetPlayerName());
```

Comment:
I'm worried that this will make it difficult to enforce unauth'd players

Reply:
You're right, but it seemed more annoying to mess with the infraction system to handle unsaved punishments. Especially considering this should be pretty rare.